styleguide/pane.scss

```diff
@@ -7,6 +7,11 @@
 $pane-margin: 30vh;
 $easeOutExpo: cubic-bezier(.190, 1.000, .220, 1.000);
 
+.hide,
+.kiln-pane-template-elements {
+  display: none !important; // Always hide your skeletons and other hidden things
```

Comment:
We should have "Utility Classes". We also need to get organized on how we are using styles. Like – how do we use multiclass selector styles to describe states? How would we name such things, etc. I know the CSS stuff is maybe not top of list but we need to get all of this together logically, especially if we introduce user-addressable-component-level-styles.